src/tiledimage.js

```diff
@@ -533,6 +549,32 @@ $.extend($.TiledImage.prototype, $.EventSource.prototype, /** @lends OpenSeadrag
                 OpenSeadragon.getElementPosition( this.viewer.element ));
     },
 
+    // private
+    // Convert rectangle in tiled image coordinates to viewport coordinates.
+    _tiledImageToViewportRectangle: function(rect) {
+        var scale = this._scaleSpring.current.value;
```

Comment:
Hmm... and y between 0 and aspect ratio?

Seems like it would be nice not to introduce yet another coordinate system, but if it's needed, at least it's private. Even so, maybe there's a better name for this coordinate system than "tiledImage" to distinguish it from the "image" coordinate system?